LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/UI/UIMapper.swift

```diff
@@ -74,21 +44,39 @@ struct MapperToUI {
                                              totalScans: totalScans,
                                              scannedBrokers: partiallyScannedBrokers)
 
-        let matches = mapMatchesToUI(brokerProfileQueryData)
+        let matches = mapMatchesToUI(withoutDeprecated)
 
         return .init(resultsFound: matches, scanProgress: scanProgress)
     }
 
     private func mapMatchesToUI(_ brokerProfileQueryData: [BrokerProfileQueryData]) -> [DBPUIDataBrokerProfileMatch] {
```

Comment:
I don't like how much this and the maintenance scans function duplicate each other. However I'm also afraid to touch them since there seem to be some subtle differences which I assume are deliberate

Reply:
Agree.